conf/agent/agent.conf

```diff
@@ -6,6 +6,10 @@ agent {
     socket_path ="/tmp/spire-agent/public/api.sock"
     trust_bundle_path = "./conf/agent/dummy_root_ca.crt"
     trust_domain = "example.org"
+    admin_socket_path = "/tmp/admin.sock"
```

Comment:
Hmm, since we're attesting the delegatee, maybe the DelegatedIdentityAPI should be served on the normal public workload API socket? @evan2645, what do you think?

Reply:
It is a good question ... I'm not sure, I think it deserves some more thought.

Workload API is secured via workload attestation, but it is always possible that a vulnerability allows the process to be manipulated in some way, or even bypassed. I think the question we should ask is "how might the workload API security fail, and would exposing an RPC of this nature there make the situation worse?"... my knee jerk answer is "probably?" Cost/benefit will be important here

Reply:
> Cost/benefit will be important here

For example .. do we expect workloads in these delegated env's to also be accessing workload api directly, thus introducing the need to inject and manage two different sockets at once